go.mod

```diff
@@ -21,10 +21,11 @@ require (
 	cloud.google.com/go/pubsub v1.17.1
 	cloud.google.com/go/storage v1.19.0
 	cloud.google.com/go/workflows v1.1.1
-	github.com/Azure/azure-amqp-common-go/v3 v3.2.2
-	github.com/Azure/azure-event-hubs-go/v3 v3.3.16
+	github.com/Azure/azure-amqp-common-go/v3 v3.2.3
+	github.com/Azure/azure-event-hubs-go/v3 v3.3.17
 	github.com/Azure/azure-sdk-for-go v61.4.0+incompatible
-	github.com/Azure/azure-sdk-for-go/sdk/azidentity v0.12.0
+	github.com/Azure/azure-sdk-for-go/sdk/azidentity v0.13.0
```

Comment:
Updating this dependency is currently problematic because `azblob` — another of our imports — doesn't yet support it. The message thrown by `golangci-lint` is cryptic, but I think it is related to that updated.

I checked what version of `azservicebus` introduced this requirement, and it turns out to be quite recent. The following command should fix things:

```
go get github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus@v0.3.2
```

Reply:
Is this a breaking change for azblob?

Reply:
Not really because we **can't** do anything with `azblob` right now. It hasn't been updated in a while and therefore doesn't support `azidentity v0.13.0`.

`azblob` was introduced when we migrated the E2E tests for Azure to this repo, but it has been causing some headaches because it's now blocking the update of `azidentity` and related packages (all the `sdk/resourcemanager/...` packages).

Reply:
Forget about it, things got resolved in #567